_layouts/home.html

```diff
@@ -2,47 +2,88 @@
 layout: base
 ---
 
-<section>
-  <div class="hero-unit" style="background-image: url('{{ page.hero-image | prepend: site.baseurl }}')">
+  <section class="usa-hero">
     <div class="usa-grid">
-      <div class="usa-width-whole hero-content">
-        <h1>{{ page.hero-text }}</h1>
-        <a class="usa-button" href="{{ page.hero-button-link | prepend: site.baseurl }}">{{ page.hero-button-text }}</a>
+      <div class=="usa-width-one-third" style="float: left;margin-right: 2em;">
```

Comment:
Looks like there's an extra `=` in here

Reply:
this explains why the css class wasn't picking up and why I to put in inline (and now can remove!) Great catch!